index.css

```diff
@@ -0,0 +1,366 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&family=Lobster&family=Roboto+Mono:wght@200;400&display=swap');
+
+*,
+*::before *::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 62.5%;
+  padding: 0;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+  background-color: #f8f8f8;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 2% 0;
+  box-shadow: 0 7px 11px 1px rgba(0, 0, 0, 0.5);
+}
+
+.header__svg {
+  position: relative;
+  left: 25px;
+  width: 11%;
+}
+
+.header__img {
+  width: 45%;
+  margin: auto 29%;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  margin: 10% 5%;
+}
+
+.hero__h2 {
+  color: #ec5242;
+  font-size: 1.5rem;
+  font-weight: 100;
+  margin-bottom: 0;
+}
+
+.hero__h1 {
+  color: #e91a03;
+  font-size: 3rem;
+  margin: 0;
+}
+
+.hero__p {
+  color: #515151;
+  border: 6px solid #fff;
+}
+
+.hero__h2-2 {
+  color: #595959;
+  font-weight: 900;
+  font-size: 1.5rem;
+  margin-bottom: 0;
+}
+
+.hero__span {
+  color: #868686;
+  font-size: 1rem;
+}
+
+.hero__date {
+  padding: 4% 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('./img/pattern.png');
+  background-repeat: repeat;
+  height: 100%;
+  width: 100%;
+  font-size: 1rem;
+  color: #fff;
+}
+
+.main__h2 {
+  font-size: 1.2rem;
+  margin: 5% 5% 1% 5%;
+}
+
+.cards-container {
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem 0;
+  gap: 5px;
+}
+
+.cards {
+  display: flex;
+  align-items: center;
+  margin: auto 3%;
+  background-color: rgba(133, 133, 133, 0.3);
+}
+
+.main__h3 {
+  color: #ff5438;
+}
+
+.cards__svg {
+  width: 20%;
+  margin: 0 5%;
+}
+
+.cards__childs {
+  padding: 1px 0.5rem;
+}
+
+.main__button {
+  font-size: 1rem;
+  font-weight: 700;
+  background-color: #ff5438;
+  color: #fff;
+  border: none;
+  padding: 1.5rem 3rem;
+  margin: 1rem 0;
+}
+
+.speakers {
+  display: flex;
+  gap: 15px;
+  padding: 4%;
+}
```

Comment:
- [ ] i can see that you tried to implement some required changes good job so far 👍  but still there is some thing that you are missing out as requested by the previous reviewer kindly consider adding the short horizontal line that comes after the speaker's name and title and separates them both from the speaker's 
- [ ] I also noticed that the width of each speaker when on a desktop screen is wider than that of the original design. Please try to ensure that the layout of your projects looks similar to that of the original design 💯